### PR TITLE
enable multisink mode in docker (non multiagent mode)

### PR DESCRIPTION
**Pull request by AlvaroVega**
Previously non multiagent mode was not usefull, since is starting all sinks in the same port (5050)

fix issue https://github.com/telefonicaid/fiware-cygnus/issues/1675

It will be enabled when:
CYGNUS_MULTIAGENT=false

- Ports will use convention 5050 mysql, 5051 mongo, 5052, ckan...
- Namemappings file will be in name_mapping_mysql.conf, name_mapping_mongo.conf, name_mapping_ckan.conf, etc
- No grouping rule support is included (soon will be removed from onpremise)

### CODE REVIEW

File: docker/cygnus-ngsi/multisink_agent.conf [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.nmi.type = com.telefonica.iot.cygnus.interceptors.NGSINameMappingsInterceptor$Builder
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.nmi.name_mappings_conf_file = /opt/apache-flume/conf/name_mappings.conf
+
+# NGSIElasticsearchSink configuration
+cygnus-ngsi.sinks.elasticsearch-sink.type = com.telefonica.iot.cygnus.sinks.NGSIElasticsearchSink
+cygnus-ngsi.sinks.elasticsearch-sink.channel = elasticsearch-channel
+cygnus-ngsi.sinks.elasticsearch-sink.elasticsearch_host = localhost
+cygnus-ngsi.sinks.elasticsearch-sink.elasticsearch_port = 9200
+cygnus-ngsi.sinks.elasticsearch-sink.ssl = false
+#cygnus-ngsi.sinks.elasticsearch-sink.index_prefix = cygnus
```
Comment (fgalan) [inline docker/cygnus-ngsi/multisink_agent.conf]: Are the commented lines correct in this file? (I think they are used as "templates" to be replaced by sed or something like that, but I'm not fully sure...)

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md [hunk 2]
```diff
@@ -156,7 +156,8 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md]: Should the fix in this PR be mentioned in CNR?

File: docker/cygnus-ngsi/multisink_agent.conf (hunk 1 above)
Comment (AlvaroVega) [reply docker/cygnus-ngsi/multisink_agent.conf]: Yes, it's a template based on previous agent.conf: https://github.com/telefonicaid/fiware-cygnus/blob/master/docker/cygnus-ngsi/agent.conf

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md (hunk 2 above)
Comment (AlvaroVega) [reply doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md]: fixed in 2b78f9ec66c23ba035202ace15cdcb73ebd13679

File: docker/cygnus-ngsi/multisink_agent.conf (hunk 1 above)
Comment (fgalan) [reply docker/cygnus-ngsi/multisink_agent.conf]: Ok. NTC

**Pull request merged**